### PR TITLE
Reduce cognitive complexity.

### PR DESCRIPTION
**Pull request by ADmad**
Update return type.

### CODE REVIEW

**Review by dereuromark — APPROVED** [review]

File: src/ORM/Table.php [hunk 1]
```diff
@@ -695,39 +695,34 @@ public function setDisplayField($field)
...
-                }
+        }
+
+        foreach ($schema->columns() as $column) {
+            $columnSchema = $schema->getColumn($column);
+            if (
+                $columnSchema &&
+                $columnSchema['null'] !== true &&
+                $columnSchema['type'] === 'string' &&
+                !preg_match('/pass|token|secret/', $column)
```
Comment (othercorey) [inline src/ORM/Table.php]:
```suggestion
                !preg_match('/pass|token|secret/i', $column)
```
This should probably be case-insensitive.

**Comment by othercorey** [comment]
Looks good as long as we're ok assuming a primary key can be displayed. Not sure what situations wouldn't work though. Don't think you have binary primary keys.

**Comment by ADmad** [comment]
@othercorey To be clear I am haven't done any behavior change, just refactored the code. Use of primary key is already present. If there are cases where it can't be displayed then it's upto the dev to specify proper field.

**Comment by othercorey** [comment]
> @othercorey To be clear I am haven't done any behavior change, just refactored the code. Use of primary key is already present. If there are cases where it can't be displayed then it's upto the dev to specify proper field.

Got it. I assumed there was a return null path that I didn't see.

**Pull request merged**